spring-vault-core/src/main/java/org/springframework/vault/core/VaultKeyValueMetadataTemplate.java

```diff
@@ -0,0 +1,82 @@
+package org.springframework.vault.core;
+
+import java.io.IOException;
+
+import org.springframework.util.Assert;
+import org.springframework.vault.VaultException;
+import org.springframework.vault.client.VaultResponses;
+import org.springframework.vault.support.VaultMetadataRequest;
+import org.springframework.vault.support.VaultMetadataResponse;
+import org.springframework.vault.support.VaultMetadataResponseWrapper;
+import org.springframework.web.client.HttpStatusCodeException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+public class VaultKeyValueMetadataTemplate implements VaultKeyValueMetadataOperations {
+
+  private final VaultOperations vaultOperations;
+
+  private static final ObjectMapper OBJECT_MAPPER;
+
+  static {
+    ObjectMapper mapper = new ObjectMapper();
+    JavaTimeModule module = new JavaTimeModule();
+    mapper.registerModule(module);
+    mapper.configure(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE, true);
+    OBJECT_MAPPER = mapper;
+  }
+
+
+  public VaultKeyValueMetadataTemplate(VaultOperations vaultOperations) {
```

Comment:
We need to pass on the path for the kv backend, similar to `VaultVersionedKeyValueTemplate`.